src/Util.jl

```diff
@@ -25,8 +21,8 @@ function get_bvn_cov{NumType <: Number}(ab::NumType, angle::NumType, scale::NumT
     #    The 2x2 covariance matrix parameterized by the inputs.
 
     #@assert -pi/2 <= angle < pi/2
-    @assert 0 < scale
-    @assert 0 < ab <= 1.
+    #@assert 0 < scale
+    #@assert 0 < ab <= 1.
```

Comment:
Apartment `<` isn't compatible with the GradientNumbers from ForwardDiff. I wish it was.